engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -252,21 +246,17 @@ case class JesBackend(actorSystem: ActorSystem)
 
   override def adjustInputPaths(jobDescriptor: BackendCallJobDescriptor): CallInputs = jobDescriptor.locallyQualifiedInputs mapValues gcsPathToLocal
 
-  override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs mapValues {
-    case CallOutput(value, hash) => CallOutput(gcsPathToLocal(value), hash)
-  }
-
-  private def writeAuthenticationFile(workflow: WorkflowDescriptor) = {
+  private def writeAuthenticationFile(workflow: WorkflowDescriptor): Try[Unit] = {
     val log = workflowLogger(workflow)
 
-    generateAuthJson(dockerConf, getGcsAuthInformation(workflow)) foreach { content =>
+    generateAuthJson(dockerConf, getGcsAuthInformation(workflow)) map { content =>
 
       val path = gcsAuthFilePath(workflow)
       def upload(prev: Option[Unit]): Unit = path.writeAsJson(content)
 
       log.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
       withRetry(upload, log, s"Exception occurred while uploading auth file to $path")
-    }
+    } getOrElse Success(Unit)
```

Comment:
Technically you can stick anything in there but `()` is the idiomatic way of returning a unit